src/assets/i18n/fr.json

```diff
@@ -52,9 +52,10 @@
     "TOOLTIP_INFO": "Info - contient des informations qui peuvent être d'intérêt pour la personne lançant le test, ces messages ne sont pas des erreurs",
     "TOOLTIP_NOTICE": "Notice - destiné à informer la personne en charge de la zone, ne correspond pas nécessairement à un problème de configuration",
     "TOOLTIP_WARNING": "Warning - destiné aux erreurs détectées dans certains cas d'usage spécifiques",
+    "Test completed": null,
```

Comment:
Should this not be given a French translation?

Reply:
good catch, I've added the translation